kernel/configs/x86_64_kata_kvm_4.14.x

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86_64 4.14.22 Kernel Configuration
+# Linux/x86 4.14.49 Kernel Configuration
```

Comment:
the latest is 4.14.55 now.

Reply:
I don't think we ever use the version field here. Just a minor change that should be ignored in all of our scripts.